SmartDeviceLink/public/SDLChoiceSet.m

```diff
@@ -67,30 +69,7 @@ - (instancetype)initWithTitle:(NSString *)title delegate:(id<SDLChoiceSetDelegat
         return nil;
     }
 
-    NSMutableSet<NSString *> *choiceTextSet = [NSMutableSet setWithCapacity:choices.count];
-    NSMutableSet<NSString *> *uniqueVoiceCommands = [NSMutableSet set];
-    NSUInteger allVoiceCommandsCount = 0;
-    NSUInteger choiceCellWithVoiceCommandCount = 0;
-    for (SDLChoiceCell *cell in choices) {
-        [choiceTextSet addObject:cell.text];
-        if (cell.voiceCommands == nil) { continue; }
-        [uniqueVoiceCommands addObjectsFromArray:cell.voiceCommands];
-        choiceCellWithVoiceCommandCount += 1;
-        allVoiceCommandsCount += cell.voiceCommands.count;
-    }
-    if (choiceTextSet.count < choices.count) {
-        SDLLogE(@"Attempted to create a choice set with duplicate cell text. Cell text must be unique. The choice set will not be set.");
-        return nil;
-    }
-
-    // All or none of the choices must have VR commands
-    if ((choiceCellWithVoiceCommandCount > 0 && choiceCellWithVoiceCommandCount < choices.count)) {
-        SDLLogE(@"If using voice recognition commands, all of the choice set cells must have unique VR commands. There are %lu cells with unique voice commands and %lu total cells. The choice set will not be set.", (unsigned long)choiceCellWithVoiceCommandCount, (unsigned long)choices.count);
-        return nil;
-    }
-    // All the VR commands must be unique
-    if (uniqueVoiceCommands.count < allVoiceCommandsCount) {
-        SDLLogE(@"If using voice recognition commands, all VR commands must be unique. There are %lu unique VR commands and %lu VR commands. The choice set will not be set.", (unsigned long)uniqueVoiceCommands.count, (unsigned long)allVoiceCommandsCount);
+    if (![self sdl_choiceCellsAreUnique:choices]) {
         return nil;
     }
```

Comment:
```suggestion
    if (![self sdl_choiceCellsAreUnique:choices]) { return nil; }
```

If statements that return in one line are placed into one line as general style for this project.